railties/test/application/rake_test.rb

```diff
@@ -126,6 +126,47 @@ def test_rake_routes_with_controller_environment
       assert_equal "Prefix Verb URI Pattern     Controller#Action\n  cart GET  /cart(.:format) cart#show\n", output
     end
 
+    def test_rake_routes_with_controller_environment_is_deprecated
```

Comment:
We shouldn't have two tests for the same thing when it's going away soon. Also I think it would actually be better to just bend the above test to add `assert_deprecated`.